src/tiledimage.js

```diff
@@ -1,36 +1,36 @@
 /*
- * OpenSeadragon - TiledImage
- *
- * Copyright (C) 2009 CodePlex Foundation
- * Copyright (C) 2010-2013 OpenSeadragon contributors
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions are
- * met:
- *
- * - Redistributions of source code must retain the above copyright notice,
- *   this list of conditions and the following disclaimer.
- *
- * - Redistributions in binary form must reproduce the above copyright
- *   notice, this list of conditions and the following disclaimer in the
- *   documentation and/or other materials provided with the distribution.
- *
- * - Neither the name of CodePlex Foundation nor the names of its
- *   contributors may be used to endorse or promote products derived from
- *   this software without specific prior written permission.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
- * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
- * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
- * A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT
- * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
- * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
- * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
- * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
- * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
- * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
- * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
+* OpenSeadragon - TiledImage
```

Comment:
I guess this isn't a big deal, but we do prefer this block of comments to be indented one space so the asterisks line up. That is how all the other files are, so it would be nice to keep this file the same.

Reply:
Yes, this is my IDe's doings, sometimes I don't notice it changed formatting somewhere :/

Reply:
I also thought about splitting PRs, but I did not know how to do it from the same source since all pushes end up in the same PR, should learn git API more properly..

Reply:
And it seems that the last test about pixels was something browser-related, in non-interactive mode, all tests are passing again.

Reply:
To make multiple PRs, you need to have a different branch for each one, so it helps to have planned that ahead of time. In general, I recommend making a new branch for each PR anyway, rather than making your PR off of master... That helps if you need to grab the latest from the upstream repository in order to fix merge conflicts or something. Anyway, stuff to do next time :)